SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -207,25 +207,19 @@ void GenerateSubobjectSchemaForActorIncludes(FCodeWriter& Writer, TSharedPtr<FUn
 {
 	TSet<UStruct*> AlreadyImported;
 
-	for (auto& PropertyPair : TypeInfo->Properties)
+	for (const auto& OffsetToSubobject : GetAllSubobjects(TypeInfo))
 	{
-		GDK_PROPERTY(Property)* Property = PropertyPair.Key;
-		GDK_PROPERTY(ObjectProperty)* ObjectProperty = GDK_CASTFIELD<GDK_PROPERTY(ObjectProperty)>(Property);
+		const TSharedPtr<FUnrealType>& PropertyTypeInfo = OffsetToSubobject.Value;
 
-		TSharedPtr<FUnrealType>& PropertyTypeInfo = PropertyPair.Value->Type;
+		UObject* Value = PropertyTypeInfo->Object;
 
-		if (ObjectProperty && PropertyTypeInfo.IsValid())
+		if (Value != nullptr && IsSupportedClass(Value->GetClass()))
```

Comment:
Looks like this is already checked inside `GetAllSubobjects`, so this can be removed here.